src/main/java/me/apemanzilla/krist/turbokrist/miners/GPUMiner.java

```diff
@@ -69,7 +69,7 @@
 		}
 		this.kernel = program.createKernel("krist_miner_basic");
 		Pointer<Byte> addressPtr = Pointer.allocateBytes(10).order(context.getByteOrder());
-		byte[] addressBytes = MinerUtils.getBytes(options.getKristAddress().getName());
+		byte[] addressBytes = MinerUtils.getBytes(options.getMiningAddress());
```

Comment:
Why do we even have `MinerUtils.getBytes`? Just do `options.getMiningAddress().getBytes("UTF-8")`.

Reply:
hey!! not my code

Reply:
i'd change this in another PR - just in case it causes breaking changes

Reply:
Alright sure.